plugins/cpp_metrics/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+# The C++ metrics plugin depends on the C++ plugin's components to be built.
+
+if ("${cpp_PLUGIN_DIR}" STREQUAL "")
+  # Use SEND_ERROR here so a build file is not generated at the end.
+  # CodeCompass might use a lot of plugins and produce a lengthy build, so
+  # a warning at configure time would easily be missed by the users.
+  message(SEND_ERROR
+    "C++ Metrics plugin found without C++ plugin in the plugins directory.")
+endif()
+
+add_subdirectory(model)
+add_subdirectory(parser)
+add_subdirectory(service)
+
+install_webplugin(webgui)
```

Comment:
The plugin does not have a `webgui` subdir (yet), this should be removed or commented out.

Reply:
There is an empty webgui subdir, so I leave this in.

Reply:
Empty directories are not added by Git. You either need to add a file into it (e.g. a `.gitkeep` file) or please remove / comment this line in the build configuration.

Reply:
Fixed in 667361582fcf5b58c66ed660fe37a2aa57b5fa59.